demos/tutorials/6_fluid_tank.ipynb

```diff
@@ -0,0 +1,280 @@
+        "\n",
+        "The fluid container is a cylinder of radius $R_\\mathrm{tank}$ and height $h_\\mathrm{tank}$. The container is initially partially filled. New fluid is injected with velocity $v_\\mathrm{inlet}$ into the tank from a circular inlet at the top of the container with arbitrary coordinates $(x_\\mathrm{inlet}, y_\\mathrm{inlet})$. The fluid exits the tank from a cylindrical outlet at the bottom of the container.\n",
+        "\n",
+        "In this tutorial, we perform simulations of this scenario for varying $x_\\mathrm{inlet}$, and show that it has an important impact on the average fluid velocity inside the tank. We thus illustrate that Inductiva API can be used for streamlined experimentation with simulations, including in optimization pipelines."
```

Comment:
But isn't our goal an optimal design? Hence, our focus should be immediately maximizing the average fluid velocity and verifying its impact in terms the of the inlet.

Maybe we should point out that it is enough to only vary the x position of the inlet due to the rotational symmetry.

Reply:
> But isn't our goal an optimal design? Hence, our focus should be immediately maximizing the average fluid velocity and verifying its impact in terms the of the inlet.

Not sure what you mean here. Can you elaborate, please?

Reply:
> Maybe we should point out that it is enough to only vary the x position of the inlet due to the rotational symmetry.

I don't want to get into these technical details. I added a sentence to explain that our goal is to illustrate how a single example parameter affects a metric of interest, though. In other words, we chose one parameter to vary but could have chosen a number of others.

Reply:
This is related with the story we want to tell in the notebook. If it is more about experimental studying something then I think your sentence here is perfect, but if it is more about optimizing/optimal design that the goal is already to maximize the avg. fluid velocity (since it may influence other interesting properties) then the sentence should be reverted, because we are choosing to do it just with the inlet position. We could also do it with the outlet at the same time.

Reply:
> I don't want to get into these technical details. I added a sentence to explain that our goal is to illustrate how a single example parameter affects a metric of interest, though. In other words, we chose one parameter to vary but could have chosen a number of others.

Fair enough, my mind just immediately went to why just use the x-position instead of both. So I though of clarifying it.

Reply:
Yep, that's a great point. We were missing a sentence that said "this is just an example parameter exploration, we could have picked any other(s)".